### PR TITLE
Revert all LWMA3 Changes - Not in use

### PR DESCRIPTION
**Pull request by brandonlehmann**

### CODE REVIEW

File: src/CryptoNoteCore/Difficulty.h [hunk 1]
```diff
@@ -5,15 +5,12 @@
...
 
-uint64_t nextDifficultyV6(std::vector<uint64_t> timestamps, std::vector<uint64_t> cumulativeDifficulties);
-
 uint64_t nextDifficultyV5(std::vector<uint64_t> timestamps, std::vector<uint64_t> cumulativeDifficulties);
 
 uint64_t nextDifficultyV4(std::vector<uint64_t> timestamps, std::vector<uint64_t> cumulativeDifficulties);
 
 uint64_t nextDifficultyV3(std::vector<uint64_t> timestamps, std::vector<uint64_t> cumulativeDifficulties);
 
-/* TODO: This has been added in the stdlib in c++17 */
```
Comment (zpalmtree) [inline src/CryptoNoteCore/Difficulty.h]: ?

Comment (brandonlehmann) [reply src/CryptoNoteCore/Difficulty.h]: These are straight reverts. You added that, why?

Comment (zpalmtree) [reply src/CryptoNoteCore/Difficulty.h]: Don't need this clamp anymore when we can use the stdlib one, but if it's just a straight revert then fine

Comment (brandonlehmann) [reply src/CryptoNoteCore/Difficulty.h]: https://github.com/turtlecoin/turtlecoin/commit/1eab20ffb619c2cf24327c57ec0180bd9bf72638#diff-ca7f1bf42b1a2086fe072fdefa8c6decR16

Straight revert :)

**Pull request merged**